src/runtime_src/core/edge/sw_em/generic_pcie_hal2/shim.cxx

```diff
@@ -1855,6 +1857,63 @@ int CpuemShim::xclCloseContext(const uuid_t xclbinId, unsigned int ipIndex) cons
   return 0;
 }
 
+
+int CpuemShim::xclIPName2Index(const char *name)
```

Comment:
Isn't this function still the same as common_em version?

Reply:
Moved to to common_em.